src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -3722,10 +3722,10 @@ unsafe void M()
                 // (8,24): error CS0119: 'Action' is a type, which is not valid in the given context
                 //         Action ptr1 = (Action)&M;
                 Diagnostic(ErrorCode.ERR_BadSKunknown, "Action").WithArguments("System.Action", "type").WithLocation(8, 24),
-                // (9,23): error CS8800: Cannot convert a & method group 'M' to delegate type 'M'.
+                // (9,23): error CS8800: Cannot convert & method group 'M' to delegate type 'M'.
```

Comment:
Not to bikeshed on the message too much but I admit seeing a & with spaces around it my brain just parsed as a shortcut for "and" and this reads as "cannot convert and method group" and it was pretty confusing. "Cannot convert the address of a method group 'M'" or consider removing the space after & to make it clear it's the operator?

Reply:
Done